docs/content/policy-reference.md

```diff
@@ -842,9 +842,10 @@ Note that the opa executable will need access to the timezone files in the envir
 ### Graphs
 
 | Built-in | Description | Wasm Support |
-| ------- |-------------|---------------|
+| ------- |-------------|--------------|
 | <span class="opa-keep-it-together">``walk(x, [path, value])``</span> | ``walk`` is a relation that produces ``path`` and ``value`` pairs for documents under ``x``. ``path`` is ``array`` representing a pointer to ``value`` in ``x``.  Queries can use ``walk`` to traverse documents nested under ``x`` (recursively). | ✅ |
 | <span class="opa-keep-it-together">``output := graph.reachable(graph, initial)``</span> | ``output`` is the set of vertices [reachable](https://en.wikipedia.org/wiki/Reachability) from the ``initial`` vertices in the directed ``graph``.  ``initial`` is a set or array of vertices, and ``graph`` is an object containing a set or array of neighboring vertices. | ✅ |
+| <span class="opa-keep-it-together">``output := graph.reachable_paths(graph, initial)``</span> | ``output`` is the set of arrays of paths [reachable](https://en.wikipedia.org/wiki/Reachability) from the ``initial`` vertices in the directed ``graph``.  ``initial`` is a set or array of paths, and ``graph`` is an object containing a set or array of root vertices.    | `SDK-dependent` |
```

Comment:
[nit] Sorry, hadn't noticed this before. Can we drop the link? I think knowing about "reachability" in a graph context is OK to assume; also this is the first link a search engine would give you if you've never heard of it before... What do you think?

Reply:
Agreed. Dropping the link.